recipe/meta.yaml

```diff
@@ -19,21 +19,21 @@ source:
     - h5py_win_setup_build.py.patch
 
 build:
-  number: 2
+  number: 3
 
 requirements:
   build:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
     - cython
     - six
     - pkgconfig
 
   run:
     - python
     - numpy x.x
-    - hdf5
+    - hdf5 1.8*
```

Comment:
I think we can live with it. My hope is we rebuild with `1.8.16` once we are ready anyways.

Personally, I would rather stricter pinning here as we have found it to be a big pain point. We can always rebuild it with a different pinning when we need to.